mifosng-android/src/main/java/com/mifos/mifosxdroid/online/surveylist/SurveyListFragment.java

```diff
@@ -23,6 +24,8 @@
 import com.mifos.objects.survey.Survey;
 import com.mifos.utils.Constants;
 
+
+
```

Comment:
Don't add unnecessary new lines in your code. Before committing your code, please make sure you reformat your code in the IDE.